controllers/addons_controller.go

```diff
@@ -157,7 +173,7 @@ func (r *AddonsLayerReconciler) checkSuccess(l layers.Layer) error {
 
 	applyWasSuccessful, err := applier.ApplyWasSuccessful(ctx, l)
 	if err != nil {
-		// TODO - we might want to add some sort of error handling here
+		r.Log.Error(err, "check for apply required failed", "requestName", l.GetName())
```

Comment:
ApplyWasSuccessful serves a different purpose than ApplyIsRequired.  This log message is misleading - probably a cut-and-paste error?

Reply:
done